src/misc/sw-template.js

```diff
@@ -299,7 +299,7 @@ function WebpackServiceWorker(params) {
 }
 
 function addAllNormalized(cache, requests) {
-  return Promise.all(requests.map(fetch)).then((responses) => {
+  return Promise.all(requests.map((request) => fetch(request))).then((responses) => {
```

Comment:
Okay, I see, because of index passed as a second argument to the `fetch`. Does it cause any problem at a moment?